### ISSUE TITLE
Can that be installed through Conda? And will it work with jupyter-console?

### ISSUE THREAD

**Issue by ubaldot** [mathworks/jupyter-matlab-proxy#27]
I am wondering if it is possible to install it through conda?

And, once installed, if I can run the kernel with `$ jupyter-console --kernel=matlab`

**Comment by ubaldot** [comment]
Thanks for the reply. 
Could we keep this issue open until the kernel will be ready to be opened also by jupyter-console?

**Comment by ubaldot** [comment]
Hello! Is there any update for this issue?

**Comment by prabhakk-mw** [comment]
Hi @ubaldot, 
Thanks for circling back, @krisctl  is currently working on putting the infrastructure in place to enable this capability. 
We expect this to be available for use in the coming weeks, and will update this issue when its ready.
Thanks!

**Comment by ubaldot** [comment]
Thanks! You mean both the features conda and jupyter-console?

**Comment by prabhakk-mw** [comment]
@ubaldot , 
No, I was only referring to using the kernel from jupyter-console.

We do not plan to publish to **conda**.
Is there a workflow that is not well served via our current release strategy to PyPI?

**Comment by ubaldot** [comment]
Ah ok, thanks! Looking forward to jupyter console integration! 

There are many differences between PyPI and conda-forge for instance, with the second preferred especially when it comes into scientific work, plus several other things such as making sanity checks of the environment before installing packages and so on. You can also pack binaries with conda not just python code. Conda-forge is taking traction fairly a lot. 
My suggestion is you guys to check conda-forge. It's not just hype. Once you have a package on PyPI it is rather straightforward to make a conda package as well. :)

**Comment by zhangxiaoxing** [comment]
I agree that familiarity with both Conda and pip is essential for anyone utilizing Conda in the first place, as it enables comprehensive package management. While leveraging both tools can enhance versatility, I'd like to highlight that deploying projects solely through Conda often projects a more polished and professional image. In contrast, relying on a mix of Conda and post-hoc adjustments via pip might inadvertently convey a less refined, 'rapid development' aesthetic – akin to a prototype rather than a finished product. Just my 2 cents.

**Comment by krisctl** [comment]
This restriction is now lifted. One can use `jupyter-console` with `MATLAB Kernel` as shown in the demo below. 

One caveat to note is that licensing needs to be done before hand , either by using environment variables such as `MWI_USE_EXISTING_LICENSE`, `MLM_LICENSE_FILE` (Refer [this](https://github.com/mathworks/matlab-proxy/blob/main/Advanced-Usage.md) for more information) or by licensing via Jupyter in notebook or JupyterLab.

https://github.com/user-attachments/assets/97a578ea-9936-4577-80c5-c51dd94a7371
